mtp_api/apps/core/fixtures/test_file.txt

```diff
@@ -0,0 +1 @@
+is awesome
```

Comment:
(see below first) i suspect this file won't be needed because most of `test_upload_dump_for_ap.py` will go away